api/app/Services/SpiraValidator.php

```diff
@@ -11,7 +19,29 @@ public function validateFloat($attribute, $value, $parameters)
 
     public function validateUuid($attribute, $value, $parameters)
     {
-        return preg_match('/^\{?[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}\}?$/', $value);
+        return Uuid::isValid($value);
+    }
+
+    public function validateCreateOnly($attribute, $value, $parameters)
+    {
+        $original = $this->model->getOriginal($attribute);
+        if (is_null($original)) {
+            return true;
+        }
+
+        if ($original == $value) {
+            return true;
+        }
+
+        return false;
+    }
```

Comment:
I'm surprised laravel doesn't have this option. I guess it is because they intend validation to be defined on a per controller method basis